### DISCUSSION TITLE
有没有基于morjs的专用组件库呢？

### DISCUSSION THREAD

**Discussion by adoin** [eleme/morjs#171] [Q&A] [answered]
能基本确保多种编译结果正常运行

**Comment by hwaphon** [comment, marked as answer]
目前，我们尚未构建一个统一的、面向外部的通用组件库，且短期内没有规划建设此类库。我们已经对社区中广泛使用的组件库进行了梳理和适配，具体信息请参阅 [MorJS 使用社区组件库指南](https://mor.ele.me/guides/advance/use-community-component) 。去年，我们对vant-weapp和td-design两个组件库进行了兼容性适配，保障了一些主要组件的可用性。但鉴于组件库的多样性和跨端语法的复杂性，我们未能实现全部组件的适配。您可以根据指南中的步骤尝试使用，看是否能满足您对组件的需求。如果您遇到组件使用上的问题，欢迎加入我们的钉钉群进行反馈。

同时，我们也鼓励大家参考 [五分钟教会你如何让小程序组件库支持多端](https://mor.ele.me/guides/advance/learn-create-component-library)，利用MorJS为社区构建一个跨端的通用组件库，为社区成员提供更多便利。

**Reply by coalo** [reply]
你好，现在官网的500人钉钉群已满，是否有其他群可以加入呢
